app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/model/Book.java

```diff
@@ -18,112 +24,195 @@
     /**
      * No args constructor to maintain compatibility
      * with Firebase deserializer
-     * DO NOT USE
+     * TO BE USED BY BOOK CONTROLLER ONLY
```

Comment:
I don't think this is used by the book controller. Why is this comment here?

Reply:
Do you call Book() in the book controller?

Reply:
Implicity yes, through firebase.

Reply:
Then it is not used by the book controller. People might be confused by this

Reply:
`TO BE USED BY FIREBASE ONLY` then? But only the book controller uses firebase in a way that will cause this constructor to be used. So technically it's the book controller that uses this constructor.

Reply:
If the book controller does not call directly. If someone else edits book controller they may become confused. I think "TO BE USED BY FIREBASE ONLY" would be best